### PR TITLE
Jira Connector + Release Notes

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/en/docs/developerportal/collaborate/connect-jira.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+tags: ["app", "jira", "project management", "scrum master", "general settings", Developer Portal", "Scrum Master"]
+---
+
+{{% alert color="info" %}}
+This feature is in Public Beta.
+{{% /alert %}}
+
+## 1 Introduction
+
+If you are a Scrum Master of a Mendix app, you can link your app to Jira if you manage your project there. Once your app is connected to Jira, you have the option to associate your commit in Studio Pro to one or more stories created in Jira.
```
Comment (ghost) [inline content/en/docs/developerportal/collaborate/connect-jira.md]: Can we put "commit" in plural?

"associate your commit" > "associate your commits"

File: content/en/docs/developerportal/collaborate/connect-jira.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+## 1 Introduction
+
+If you are a Scrum Master of a Mendix app, you can link your app to Jira if you manage your project there. Once your app is connected to Jira, you have the option to associate your commit in Studio Pro to one or more stories created in Jira. 
+
+## 2 Features
+
+* Supports connecting your Mendix app to Jira
+* Enables you to migrate open stories (stories in any uncompleted Sprint or backlog, with the status "to-do" or "running") from the Developer Portal to Jira
+* Allows you to associate your commits in Studio Pro (version 7.19 or above) to Jira stories
+* Supports [Jira Cloud platform REST API version 3](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/#about)
```
Comment (ghost) [inline content/en/docs/developerportal/collaborate/connect-jira.md]: On second thought, I think we should remove this line. I don't think we should publish the inner workings of the Jira Connector.

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 3]
```diff
@@ -17,6 +17,12 @@ To see the current status of the Mendix Developer Portal and Control Center, see
 
 ## 2023
 
+### June 28th, 2023
```
Comment (ghost) [inline content/en/docs/releasenotes/developer-portal/_index.md]: We intend to release on June 22

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 4]
```diff
@@ -17,6 +17,12 @@ To see the current status of the Mendix Developer Portal and Control Center, see
...
+
+#### New Features
+
+* We have introduced [Jira connector](/developerportal/collaborate/jira-connector/) in the Developer Portal, which enables you to connect your Mendix apps to your Jira board.
```
Comment (ghost) [inline content/en/docs/releasenotes/developer-portal/_index.md]: Our PM has agreed on the following text:

Jira is here! You can now assign your Studio Pro commit to user stories in your Jira board. We offer this functionality as a first public beta version and welcome your feedback. Read all about the features and how to set it up [here](/developerportal/collaborate/jira-connector/)

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 5]
```diff
@@ -17,6 +17,12 @@ To see the current status of the Mendix Developer Portal and Control Center, see
...
+
```
Comment (ghost) [inline content/en/docs/releasenotes/developer-portal/_index.md:25]: We have the following additional release notes:

* We simplified the [Project invitation flow](/developerportal/collaborate/team/#inviting). If someone from your own company invites you to a Project that also belongs to your company, you will no longer have to accept the pending invitation. Instead you will be immediately added to the Project team
* You can now see everyone who was invited to the project and by whom. You will find this information on the [Project History](/developerportal/collaborate/general-settings/#history) tab
* We had a close look at the messages we post on the [Team History](/developerportal/collaborate/team/#managing) page and revised some of them to make them more readable
* You can now see all pending invitations to your project's team on the [Team](/developerportal/collaborate/team/#managing) page
* We fixed an issue where, under certain circumstances, the project logo was not updated in Studio Pro after you changed it in the [General Settings](/developerportal/collaborate/general-settings/) page

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/developer-portal/_index.md:25]: "project" > "app"

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/developer-portal/_index.md:25]: If the Jira connector is Public Beta, this should be stated in the RN

Comment (ghost) [reply content/en/docs/releasenotes/developer-portal/_index.md:25]: The Jira Connector is in Public Beta and **not** (yet) GA.

The Jira Connector breaks the functionality to change the status of a story in Studio Pro. As long as we have no solution for that, we mark the connector as Public Beta.

**Pull request merged**